frontend/src/component/common/UserAvatar/UserAvatar.tsx

```diff
@@ -19,6 +21,27 @@ const StyledAvatar = styled(Avatar)(({ theme }) => ({
     fontWeight: theme.fontWeight.bold,
 }));
 
+const StyledTooltip = styled(Box)(({ theme }) => ({
+    display: 'flex',
+    gap: theme.spacing(2),
+}));
+
+const StyledTooltipAvatar = styled(Avatar)(({ theme }) => ({
+    width: theme.spacing(5),
+    height: theme.spacing(5),
+    margin: 'auto',
```

Comment:
out of curiosity: what's the margin auto doing here? Does ...  does the avatar have some built-in margins?

Reply:
Not sure, was a copy paste from the existing styled avatar component.